parsers/test/test_config.py

```diff
@@ -1,45 +1,25 @@
 """Tests for config/__init__.py."""
 
-from electricitymap.contrib.config import emission_factors
+import pytest
 
+from electricitymap.contrib.config import emission_factors
 
-def test_emission_factors():
-    """Test that emission_factors handles yearly defaults correctly."""
 
-    # KR - no override
-    expected = {
-        "battery charge": 0,
-        "battery discharge": 421.94,
-        "biomass": 230,
-        "coal": 820,
-        "gas": 490,
-        "geothermal": 38,
-        "hydro": 24,
-        "hydro charge": 0,
-        "hydro discharge": 421.94,
-        "nuclear": 12,
-        "oil": 650,
-        "solar": 45,
-        "unknown": 644,
-        "wind": 11,
+@pytest.fixture
+def emission_factor_result(zone_key):
+    """Fixture to generate emission factor result in the expected format."""
+    return {
+        "emissionFactors": {
+            "defaults": {
+                key: {"value": value, "source": "IPCC 2014"}
+                for key, value in emission_factors(zone_key).items()
+            },
+            "zoneOverrides": {},
+        }
```

Comment:
I think it would be fine to test the whole thing (not just the default values) but we can keep it like this.